docs/FAQ/FAQ-Integrations.md

```diff
@@ -84,3 +85,154 @@ function get_order_id_from_custom_order_style($orderid){
 add_filter('woocommerce_order_id_from_number', 'get_order_id_from_custom_order_style', 1);
 ```
 </details>
+
+### How to configure Additional Token Support
+
+With using the additional token configuration you will be able to have separate payment methods for each configured Currency, Asset, Altcoin or Token. This means you can have separate payment methods for BTC, Lightning Network, LTC, ETH (and ERC20 tokens), Liquid assets, ... you name it. This allows you to issue and use [Liquid Assets](https://help.blockstream.com/hc/en-us/articles/900001543146-What-are-Liquid-assets-) as coupons or vouchers, see more details below.)
+
+#### Use cases
+- give away products for free via promotional tokens
+- allow discounts for certain payment methods (tokens)
+- restrict products to certain payment methods (tokens)
+- restrict payment methods (tokens) on shipping zones
+- and many more, see examples below
+
+#### Requirements
+- all tokens you configure on the WooCommerce side need to be available in your store on BTCPay Server side
+- to use promotional tokens you need to have [Liquid Assets plugin](https://https://github.com/btcpayserver/btcpayserver-plugins) installed on BTCPay Server
```

Comment:
The `Liquid Assets plugin` link doesn't work.
Might be a private repo ? Would be surprised, but maybe.

Reply:
Never mind, i'm a dumbass.
Here's the fix:

```suggestion
- to use promotional tokens you need to have [Liquid Assets plugin](https://github.com/btcpayserver/btcpayserver-plugins) installed on BTCPay Server
```

Reply:
Nice catch 👍